ChipInput.vue:  rootElement.value can be undefined, use nullcheck

daphnesmit opened the pull request:
The rootElement.value is assumed to always exist, leading to a runtime error when it is undefined, particularly when navigating between views (using popstate + autofocus). The fix involves adding a null check for rootElement.value to prevent the error during the focusOut event after unmounting.

Reproduction: https://n2ts7j.csb.app/

egardner commented on .editorconfig:
```diff
@@ -0,0 +1,10 @@
+
+[*.{js,json,yml}]
+charset = utf-8
+indent_style = space
```
Ok! We are changing so many other things that I'm willing to just leave spaces vs tabs alone. Would you prefer I remove this file or set it to tabs?

tzhelyazkova commented on README.md:
```diff
@@ -1,2 +1,27 @@
+  file to manage scripts, dependencies, etc. To add a dependency for a specific
+  package, run `yarn workspace vue-components add -D eslint`. To run workspace-specific
+  commands, use `yarn workspace components build`, etc.
+* **Zero-installs**: The project also uses Yarn's
```
what an awesome feature

daphnesmit committed:
ChipInput.vue: 

The rootElement.value is assumed to always exist, leading to a runtime error when it is undefined, particularly when navigating between views (using popstate + autofocus). The fix involves adding a null check for rootElement.value to prevent the error during the focusOut event after unmounting.

Reproduction: https://n2ts7j.csb.app/
